pyproject.toml

```diff
@@ -1,16 +1,76 @@
+# Reference: https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#static-vs-dynamic-metadata
+
 [build-system]
 requires = ["setuptools>=45", "wheel", "setuptools_scm[toml]>=6.0"]
 build-backend = "setuptools.build_meta"
 
 
+[project]
+# Basic information
+name = "gettsim"
+version = "0.5.1"
```

Comment:
This 
1. is outdated
2. should be handled by the section:

```toml
[tool.setuptools_scm]
write_to = "src/_gettsim/_version.py"
```

Reply:
The build fails without a version.

Reply:
`dynamic = ["version"]` did the trick, though.